src/app/search/search.component.ts

```diff
@@ -410,6 +421,41 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  /**
+   * Updates the results table when there is no search term
+   * Queries for 100 tools and 100 workflows
+   *
+   * @param {string} value the elastic search query
+   * @memberof SearchComponent
+   */
+  updateResultsTableSeparately(value: string) {
+    let toolHits;
+    let workflowHits;
+    const numOfResults = 201;
```

Comment:
move the constant so that https://github.com/dockstore/dockstore-ui2/pull/1110/files#diff-f92f57d57ef2daf0a8070240bc9d165e4a2a9a51285247c4b1941633bdeac556R356 can use it too